libs/game/info.ts

```diff
@@ -383,6 +383,26 @@ namespace info {
         infoState.countdownExpired = true;
     }
 
+    /**
+     * Get the value of the current count down
+     */
+    //% block="countdown"
+    //% blockId=gamegetcountdown
+    //% group="Countdown"
+    export function getCountdown() {
+        return (infoState.gameEnd - game.currentScene().millis()) / 1000;
```

Comment:
Can you add weights to these two so the order matches score / life - should be getter first, then setter, then change by,  and then stop / on countdown end

Reply:
Yes, definitely. I noticed that the weights for life / score are all different (ie score is in the 90s while life is in the 80s). Why do the different groups have to be in different ranges?

Reply:
No real reason, just historical - I don't think we had groups in this category at first and when the groups were added no one saw a reason to change the existing weights.

For what it's worth I'll sometimes still keep that pattern just so it's easier to visualize without accounting for the group - even if it doesn't matter cross group it can be helpful when laying out an entire group